crucible/src/Lang/Crucible/Backend.hs

```diff
@@ -650,6 +655,10 @@ runCHC bak uninterp_inv_fns  = liftIO $ do
     Sat sub -> return sub
     Unsat{} -> fail "Prover returned Infeasible"
     Unknown -> fail "Prover returned Fail"
+  -- CVC5.runCVC5SyGuS sym logData uninterp_inv_fns implications >>= \case
+  --   Sat sub -> return sub
+  --   Unsat{} -> fail "Prover returned Infeasible"
+  --   Unknown -> fail "Prover returned Fail"
```

Comment:
Needs to have some option to select which solver to use (other than commenting and recompiling as used here).